src/duckdb/write.rs

```diff
@@ -392,31 +398,64 @@ impl DisplayAs for DuckDBDataSink {
 fn write_to_table(
     table: &TableCreator,
     tx: &Transaction<'_>,
-    mut data_batches: Receiver<RecordBatch>,
+    schema: SchemaRef,
+    data_batches: Receiver<RecordBatch>,
 ) -> datafusion::common::Result<u64> {
-    let mut num_rows = 0;
-    let mut appender = tx
-        .appender(&table.table_name().to_string())
-        .context(super::UnableToGetAppenderToDuckDBTableSnafu)
+    let stream = FFI_ArrowArrayStream::new(Box::new(RecordBatchReaderFromStream::new(
+        data_batches,
+        schema,
+    )));
+
+    let current_ts = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .context(super::UnableToGetSystemTimeSnafu)
+        .map_err(to_datafusion_error)?
+        .as_millis();
+
+    let view_name = format!("__scan_{}_{current_ts}", table.table_name());
```

Comment:
```suggestion
    let view_name = format!("__scan_{}_{current_ts}", table.table_name());
```

`table.table_name()` will be the internal table name with a unix timestamp, if it is an internal table. Might want `table.definition_name()`, for the name from the table definition only?

Reply:
It shouldn't matter - but I'd like to keep it as the internal name, since it represents what we are ingesting into.